packages/utils/src/media/CanvasRenderTarget.ts

```diff
@@ -69,27 +75,43 @@ export class CanvasRenderTarget
      * The width of the canvas buffer in pixels.
      * @member {number}
      */
-    get width(): number
+    get width(): number | null
     {
-        return this.canvas.width;
+        if (this.canvas !== null)
+        {
+            return this.canvas.width;
+        }
+
+        return null;
     }
 
-    set width(val: number)
+    set width(val: number | null)
```

Comment:
`set width()` should not accept `null` in my opinion. Also I think it should throw when `this.canvas === null` just like I described in `get width()`.